lib/index.ts

```diff
@@ -43,7 +41,11 @@ function resultChange(result: number) {
  * with an arbitrary string, it'll produce coordinates well
  * outside of the normal range.
  */
-export function decode(str: string): Position[] {
+export function decode(str: string, precision?: number): Position[] {
+  if (typeof(precision) === 'undefined') {
+    precision = 5;
+  }
```

Comment:
This can probably be more idomatic by using a default, like

`(str: string, precision: number = 5)`